dashboard/app/models/ability.rb

```diff
@@ -105,6 +105,31 @@ def initialize(user)
         can?(:manage, section) || user.sections_as_student.include?(section)
       end
 
+      can :view_as_user, ScriptLevel do |script_level, user_to_assume|
+        can_view_as_user = false
+        can_view_as_user = true if user_to_assume.student_of?(user) ||
```

Comment:
nit: does this need to be explicitly set to false before checking?

Reply:
mmm tell me more, what would you propose?

Reply:
Oh I just meant do we need the `can_view_as_user = false` if we're reassigning it immediately after?

Reply:
I think so? So that we can return `false` on the last line of the block if none of the `true` reassignments happened?